src/services/eventNotifierSNSMessageService.ts

```diff
@@ -0,0 +1,39 @@
+import 'reflect-metadata'
+import { injectable, inject } from 'inversify'
+import TYPES from '../types'
+import { EventNotifierSNSMessage } from 'street-manager-data'
+import Logger from '../utils/logger'
+import EventNotifierSNSMessageMapper from '../mappers/eventNotifierSNSMessageMapper'
+import { SNSMessage } from 'aws-lambda'
+import { calcuateTimeDifferenceMilliseconds } from '../helpers/dateTimeHelper'
+
+export interface EventLogMessage {
+  notification_received: Date
+  event_reference: number,
+  object_reference: string,
+  time_to_notification: number
+}
+
+@injectable()
+export default class EventNotifierSNSMessageService {
+
+  public constructor(
+    @inject(TYPES.Logger) private logger: Logger,
+    @inject(TYPES.EventNotifierSNSMessageMapper) private mapper: EventNotifierSNSMessageMapper
+  ) {}
+
+  public async handleMessage(snsMessage: SNSMessage, timeNotificationreceived: Date): Promise<void> {
```

Comment:
Doesn't seem to be any async functions called within this one
```suggestion
  public handleMessage(snsMessage: SNSMessage, timeNotificationreceived: Date): void {
```

If so, the method that calls this could also have it's async/promise keyword(s) removed